Library/Formula/berkeley-db.rb

```diff
@@ -33,6 +33,26 @@ def install
       --enable-cxx
       --enable-compat185
     ]
+    if build.with? "java"
+        java_home = ""
+        if ENV.has_key?("JAVA_HOME")
+            java_home = ENV["JAVA_HOME"]
+       else
+           java_home = `/usr/libexec/java_home`.chomp
```

Comment:
That's what we do in almost most formulae. I think being consistent here is probably more important.